tests/impls/vis_sal_similarity/test_similarity_scoring.py

```diff
@@ -0,0 +1,107 @@
+from unittest import TestCase
+
+import numpy as np
+import os
+
+from xaitk_saliency.impls.vis_sal_similarity.similarityscoring import SimilarityScoring
+from xaitk_saliency import ImageSimilaritySaliencyMapGenerator
+from tests import DATA_DIR
+
+
+class TestSimilarityScoring (TestCase):
+
+    def test_init_(self) -> None:
+        """
+        Test if implementation is usable.
+        """
+        impl = SimilarityScoring()
+        assert impl.is_usable() and isinstance(impl, ImageSimilaritySaliencyMapGenerator)
+
+    def test_default_param(self) -> None:
+        """
+        Test default construction.
+        """
+        impl = SimilarityScoring()
+        assert impl.proximity_metric == 'euclidean'
+
+    def test_get_config(self) -> None:
```

Comment:
I would like to point you towards a config test helper in smqtk-core located at `from smqtk_core.configuration import configuration_test_helper`.

Reply:
Oh my, I even wrote documentation on this: https://smqtk-core.readthedocs.io/en/latest/plugins_configuration.html#help-with-writing-unit-tests-for-configurable-implementing-types

Reply:
Sweet! Used the configuration_test_helper. Please let me know what you think

Reply:
Use LGTM. I'd say update the name and description (comment) of the test however to reflect the change in what's happening here.

Reply:
Done. Thank you for the pointer.